debian/rules

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/make -f
+
+# Uncomment this to turn on verbose mode.
+#export DH_VERBOSE=1
+
+%:
+	dh $@ --with python2,sphinxdoc
+
+override_dh_clean:
+	dh_auto_clean
+	cd $(CURDIR)
+	rm -rf build/ acme/build/ letsencrypt-apache/build/ letsencrypt-nginx/build
+	rm -rf debian/python-acme/ debian/letsencrypt/ debian/letsencrypt-apache/ debian/letsencrypt-nginx/
+
+override_dh_auto_build:
+	# acme
+	cd $(CURDIR)/acme; \
+	python2.7 setup.py build; \
+	python2.7 setup.py install --single-version-externally-managed --root=$(CURDIR)/debian/python-acme --install-layout=deb
+	# letsencrypt
+	cd $(CURDIR); \
+	python2.7 setup.py build; \
+	python2.7 setup.py install --single-version-externally-managed --root=$(CURDIR)/debian/letsencrypt --install-layout=deb
+	# letsencrypt-apache
+	cd $(CURDIR)/letsencrypt-apache; \
+	python2.7 setup.py build; \
+	python2.7 setup.py install --single-version-externally-managed --root=$(CURDIR)/debian/letsencrypt-apache --install-layout=deb
+	# letsencrypt-nginx
+	cd $(CURDIR)/letsencrypt-nginx; \
+	python2.7 setup.py build; \
+	python2.7 setup.py install --single-version-externally-managed --root=$(CURDIR)/debian/letsencrypt-nginx --install-layout=deb
+	# Documentation
+	sphinx-build -N -b html docs $(CURDIR)/debian/doc-build/html
+	sphinx-build -N -b man docs $(CURDIR)/debian/doc-build/man
```

Comment:
Is there a way to split it out easily?  If not, I don't see that as a big problem.  It's definitely not a problem for -apache and -nginx.  python-acme is a little bit more questionable, but it's still not a big problem.